### PR TITLE
Cleanup of transport param and related QUIC v1 mechanics text

### PR DESCRIPTION
**Pull request by igorlord**
Minor cleanup of transport param text.

### CODE REVIEW

File: draft-ferrieuxhamchaoui-quic-lossbits.md [hunk 1]
```diff
@@ -287,23 +286,23 @@ The use of the loss bits is negotiated using a transport parameter:
 
 loss_bits (0x1055):
 
-: The loss bits transport parameter is an integer value that can be set
-  to 0 or 1 indicating the level of QL bits support.
+: The loss bits transport parameter is an integer value, encoded as a
+  variable-length integer, that can be set to 0 or 1 indicating the level of
+  QL-bits support.
```
Comment (dtikhonov) [inline draft-ferrieuxhamchaoui-quic-lossbits.md:291]: Do you want to include the changes from PR #4 here?

File: draft-ferrieuxhamchaoui-quic-lossbits.md [hunk 2]
```diff
@@ -287,23 +286,23 @@ The use of the loss bits is negotiated using a transport parameter:
...
 short packet header to QL-bits if the peer sends loss_bits=1.
 
-When loss_bits is set to 1, the sender will change R-bits to QL-bits if
-the peer sends the loss_bits transport parameter set to either 0 or 1.
+When loss_bits is set to 1, the sender will change R-bits to QL-bits if the peer
+includes the loss_bits transport parameter.
 
 A client MUST NOT use remembered value of loss_bits for 0-RTT connection.
 
-# Mechanics
```
Comment (dtikhonov) [inline draft-ferrieuxhamchaoui-quic-lossbits.md:301]: I prefer grouping the QUIC changes introduced by the extension into a single section.  Perhaps "mechanics" is not the right title -- is that why you propose dropping it?

**Review by dtikhonov — APPROVED** [review]

File: draft-ferrieuxhamchaoui-quic-lossbits.md (hunk 1 above)
Comment (igorlord) [reply draft-ferrieuxhamchaoui-quic-lossbits.md:291]: I do not think they are needed. "Supports loss bits extension" (from PR #4) is on its face obvious (how else would it know to send this TP?) and does not add anything else.

File: draft-ferrieuxhamchaoui-quic-lossbits.md (hunk 2 above)
Comment (igorlord) [reply draft-ferrieuxhamchaoui-quic-lossbits.md:301]: Yes.  I did not like "Mechanics" as a section.  If you can come up with a better title and have an intro sentence, make a PR.

Also, would you include "Transport Parameter" section as a subsection to that one?

Comment (dtikhonov) [reply draft-ferrieuxhamchaoui-quic-lossbits.md:301]: OK, I'll try to come up with something.

Good point about "Transport Parameter" section.

**Pull request merged**